src/coreclr/src/pal/src/locale/unicodedata.cpp

```diff
@@ -464,6 +464,7 @@ CONST UnicodeDataRec UnicodeData[] = {
   { 0x275, LOWER_CASE, 0x19F },
   { 0x27D, LOWER_CASE, 0x2C64 },
   { 0x280, LOWER_CASE, 0x1A6 },
+  { 0x282, LOWER_CASE, 0xA7C5 },
```

Comment:
@MichalStrehovsky, could this be a header-only or does adding .cpp in addition to .h file gives some advantage? I realize that it is an auto-generated code file, `UnicodeData[]` can still can be packed in the header (i.e. .h file can be auto-generated with some constant structs which are currently declared).
just wondering about your thoughts on .cpp vs. header-only approach in this case. :)

Reply:
Is there an advantage of header-only besides having one less file?

I generally prefer .h/.cpp split because long time ago when I did a lot of C++, precompiled headers were a PITA to deal with and from observing where C++ is heading with modules and all, people still didn't figure it out. This is a big data structure to re-parse every time the file is included. I now try to stay away from C++ as much as possible so I might not be up to date.